File: OmkarFCUBEAPI/FleetTrans/Models/BillStatementModel.cs


namespace FleetTrans.Models
{
    public class BillStatementModel
    {
        public string? MasterID { get; set; }
        public string? BillStation { get; set; }
        public string? SeriesCode { get; set; }
        public string? Bill_StmtNo { get; set; }
        public string? BillDate { get; set; }
        public string? PartyCode { get; set; }
        public string? FromDate { get; set; }
        public string? ToDate { get; set; }
        public string? FromPoint { get; set; }
        public string? ToPoint { get; set; }
        public string? Findocid { get; set; }
        public string? SuppYN { get; set; }
        public string? PartyRefNo { get; set; }
        public string? TotFreight { get; set; }
        public string? TotExtraChrg { get; set; }
        public string? TotSubTotal { get; set; }
        public string? CreditAc { get; set; }
        public string? GstType { get; set; }
        public string? SgstPct { get; set; }
        public string? SgstAmt { get; set; }
        public string? CgstPct { get; set; }
        public string? CgstAmt { get; set; }
        public string? IgstPct { get; set; }
        public string? IgstAmt { get; set; }
        public string? TotalBillAmt { get; set; }
        public string? Remarks { get; set; }
        public string? LoggedInUser { get; set; }
        public string? YearId { get; set; }
        public string? FPlace { get; set; }
        public List<BillStatementSearchModel> BillStatementListData { get; set; }


    }
}

File: OmkarFCUBEAPP/src/app/models/billstatementmodel.ts
import { Billstatementsearchmodel } from "./billstatementsearchmodel";

export class billstatementmodel {
    masterID: string = "";
    billStation: string = "";
    seriesCode: string = "";
    bill_StmtNo: string = "";
    billDate: string = "";
    fromPoint : string = "";
    toPoint : string = "";
    partyCode: string = "";
    fromDate: string = "";
    toDate: string = "";
    suppYN: string = "";
    partyRefNo: string = "";
    totFreight: string = "";
    totExtraChrg: string = "";
    totSubTotal: string = "";
    gstType: string = "";
    sgstPct: string = "";
    sgstAmt: string = "";
    cgstPct: string = "";
    cgstAmt: string = "";
    igstPct: string = "";
    igstAmt: string = "";
    totalBillAmt: string = "";
    remarks: string = "";
    yearId: string = "";
    loggedInUser: string = "";
    fPlace: string = "";
    billStatementListData: Billstatementsearchmodel[] = [];
}
